src/main/groovy/edu/oregonstate/mist/locations/core/FacilLocation.groovy

```diff
@@ -42,6 +46,10 @@ class FacilLocation extends BaseType {
     }
 
     String getPrettyCampus() {
+        if (campus == null) {
+            return null
+        }
+
         if (campus.equalsIgnoreCase("CASCADESCAMPUS")) {
             "Cascades"
```

Comment:
Could you move these string to the constants class? https://github.com/osu-mist/locations-api/pull/46/files#diff-50f2941683d1d384d7f5d7a5956bd555

Reply:
Which strings? I agree that the strings we return from this function (`"Cascades"`, `"Corvallis"`, `"HMSC"`) ought to be using the `Constants.CAMPUS_CORVALLIS` etc constants, since they are exposed to the outside world via attributes.campus.

OTOH, the strings `"CASCADESCAMPUS"` and `"OSUCAMPUS"` only appear once in the codebase, and they are specific to the FacilLocation class. I'd rather keep them close to where they are used.  No other code should depend on these values, so moving them to a global namespace would be a step backwards.

Reply:
I agree the CASCADESCAMPUS and OSUCAMPUS strings should remain in the FacilLocation class. I was thinking you could move the "Cascades", "Corvallis", "HMSC" strings to the constants class.

Reply:
That raises an interesting issue. FacilLocation uses `Capitalized` names for the campuses, but elsewhere we use `lowercase` names. I think we should use `lowercase` everywhere, but I'd rather not change the case in this PR.